components/suggestion/switch_channel_provider.jsx

```diff
@@ -115,15 +122,31 @@ class SwitchChannelSuggestion extends Suggestion {
         } else {
             icon = (
                 <div className='pull-left'>
-                    <Avatar
+                    <ProfilePicture
```

Comment:
The profile picture itself is a bit heavy and is though for places where the user itself is clickable and should show the profile popover. I'm not strictly against this, but I think we should have something like "AvatarWithStatus" component, used in the `ProfilePicture` and here, (probably in other places too).

Reply:
there is actually a conditional inside the `ProfilePicture` that renders the whole component without the action (I think it is a span).

I am not entirely sure if we should really introduce a new component. IMO the Avatar could have a conditional, that shows the status, when it is set.